easybuild/easyconfigs/l/LLVM/LLVM-3.9.0-GCC-5.4.0-2.26.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'CMakeMake'
+
+name = 'LLVM'
+version = '3.9.0'
+
+homepage = "http://llvm.org/"
+description = """The LLVM Core libraries provide a modern source- and target-independent
+ optimizer, along with code generation support for many popular CPUs
+ (as well as some less common ones!) These libraries are built around a well
+ specified code representation known as the LLVM intermediate representation
+ ("LLVM IR"). The LLVM Core libraries are well documented, and it is
+ particularly easy to invent your own language (or port an existing compiler)
+ to use LLVM as an optimizer and code generator."""
+
+toolchain = {'name': 'GCC', 'version': '5.4.0-2.26'}
+toolchainopts = {'cstd': 'gnu++11'}
+
+source_urls = ["http://llvm.org/releases/%(version)s"]
+sources = ["llvm-%(version)s.src.tar.xz"]
+
+checksums = ['66c73179da42cee1386371641241f79ded250e117a79f571bbd69e56daa48948']
+
+builddependencies = [
+    ('CMake', '3.7.1'),
+    ('Python', '2.7.12', '-bare'),
+]
+
+dependencies = [
+    ('ncurses', '6.0'),
```

Comment:
@vanzod It's not about the ncurses version...

There's an easyconfig file available for CMake 3.7.1 that uses `GCCcore/5.4.0` (and requires ncurses 6.0 as a dep, with that same toolchain).

You're listing ncurses 6.0 here as a dep, when using `GCC/5.4.0-2.26` as a toolchain, and since there's an easyconfig available for exactly that, that's also what the robot will pick.

So, the robot picks two different builds of ncurses, which leads to a conflict... That's probably a bug.

We've never run into this before because with `GCCcore` in play people don't use `GCC` as a toolchain anymore, they use `GCCcore` and list `binutils` manually as a build dep.